action.go

```diff
@@ -327,6 +327,69 @@ func (a *ActionMakeHash) Do(h *Holochain) (response interface{}, err error) {
 }
 
 //------------------------------------------------------------
+// Sign
+
+type ActionSign struct {
+	doc []byte
+}
+
+func NewSignAction(doc []byte) *ActionSign {
+	a := ActionSign{doc: doc}
+	return &a
+}
+
+func (a *ActionSign) Name() string {
+	return "sign"
+}
+
+func (a *ActionSign) Args() []Arg {
+	return []Arg{{Name: "doc", Type: EntryArg}}
+}
+
+func (a *ActionSign) Do(h *Holochain) (response interface{}, err error) {
+	var b []byte
+	//hash, err = a.entry.Sum(h.hashSpec)
```

Comment:
lets clean up this line before the final merge, I think it's left over from makeHash right?